### ISSUE TITLE
Missing OSC: How to get lua development headers

### ISSUE THREAD

**Issue by no-simpler** [mpv-player/mpv#5200]
### mpv version and platform

mpv 0f9a690 installed on **macos 10.13.1** via Homebrew with:

```
brew install mpv --HEAD --with-bundle --with-libaacs --with-libarchive \
       --with-libcaca --with-libdvdnav --with-pulseaudio --with-rubberband \
       --with-uchardet --with-vapoursynth
```

### Reproduction steps

`mpv -v --osc file.mp4`

### Expected behavior

Plays video with OSC.

### Actual behavior

```
[cplayer] Error parsing option osc (option not found)
[cplayer] Setting commandline option --osc=yes failed.
[cplayer]
[cplayer] Exiting... (Fatal error)
```

### Log file

`mpv -v --osc file.mp4`: http://sprunge.us/KVAE

`mpv -v`: http://sprunge.us/eYMC

### Note

I’ve read the FAQ:

> Install Lua (including development headers) and rebuild mpv.

Lua 5.3.4 is installed of course. How do I get Lua development headers?

Sorry to ask, I know I’m in over my head with this. _Thanks in advance if you spare a moment to help me out._

**Comment by selsta** [comment]
https://github.com/Homebrew/homebrew-core/pull/21389

**Comment by Argon-** [comment]
mpv does not support 5.3. You need 5.1 or 5.2 (both work, I've used both long enough).
To get a working version of mpv with brew, all you need to do is change one single line in the brew formula. The brew maintainers know this too but they don't want to fix it because the person who asked apparently didn't ask nice enough.
Please complain to them.

**Comment by Argon-** [comment]
To fix it locally: `brew edit mpv`
Search the line `depends_on "lua" => :recommended` and replace it with `depends_on "lua@5.1"` then rebuild from source.

**Comment by CounterPillow** [comment]
I've adjusted the FAQ page to explicitly mention the supported Lua versions.

**Comment by no-simpler** [comment]
Dear lord, this package was hard to install.

It worked, of course; replies are much appreciated.

**Comment by ilovezfs** [comment]
See my comments here: https://github.com/mpv-player/mpv/issues/5080#issuecomment-350411768

TL;DR I've merged the 5.1 PR :peace_symbol: :dove: :v:
